common/build.gradle

```diff
@@ -99,10 +99,10 @@ android {
         }
 
         // create separate code coverage tasks(for unitTests, androidTests and both Tests) for every build variant
-        def buildVariant = variant.name
-        createCoverageTask(buildVariant, unitTestType)
-        createCoverageTask(buildVariant, androidTestType)
-        createCoverageTask(buildVariant, bothTestType)
+//        def buildVariant = variant.name
+//        createCoverageTask(buildVariant, unitTestType)
+//        createCoverageTask(buildVariant, androidTestType)
+//        createCoverageTask(buildVariant, bothTestType)
```

Comment:
we should really just not create these tasks for non-debug variants.

Reply:
Yes I will follow up with Ayush to fix this in a new PR